healthcare/dicomweb_instance_retrieve.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START healthcare_dicomweb_retrieve_instance]
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	healthcare "google.golang.org/api/healthcare/v1beta1"
+)
+
+// dicomWebRetrieveInstance retrieves a specific instance.
+func dicomWebRetrieveInstance(w io.Writer, projectID, location, datasetID, dicomStoreID, dicomWebPath string, outputFile string) error {
```

Comment:
Sorry I missed this in the first review. We added a new style guide for samples and now want to include sample values for all of the arguments to functions.

See https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#function-arguments for the format.

Reply:
Done for all of the files in this PR.